person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # Getter for @id, @name and @age
+  attr_reader :id
+  attr_accessor :name, :age

```

Comment:
- [ ] For better code organization, I would recommend you move these getters and setters to the top of the class.